client/app/components/empty-state/empty-state.html

```diff
@@ -1,3 +1,4 @@
+<div ng-if="!$ctrl.isEmailVerified" class="alert alert-danger">We have sent an email with a confirmation link to your email address. Please follow the link to verify your email address. <a ng-click="verifyEmail()">Resend email</a>.</div>
```

Comment:
We're showing the EmptyState component in multiple places. Was it intentional that you used it?

Reply:
Also, maybe a bit of a less scary color for the message?

@ranbena

Reply:
re: color - another option is this:

![image](https://user-images.githubusercontent.com/289488/51083960-fd76d700-172a-11e9-9167-f2128c4c7d7e.png)

Reply:
🤢

Reply:
> We're showing the EmptyState component in multiple places. Was it intentional that you used it?

Not really. Moved to `<home-page>`.